includes/frontend/class-wcmp-checkout.php

```diff
@@ -71,6 +71,16 @@ public function enqueue_frontend_scripts()
             array_push($deps, "wcmp-checkout-fields");
         }
 
+
+        /*
+         * Show delivery options also for shipments on backorder
+         */
+        $shouldShow = $this->shouldShowDeliveryOptions();
```

Comment:
Als je die var niet hergebruikt kun je hem net zo goed direct in de if zetten, de naam van de functie is toch leesbaar genoeg

Reply:
done